public/index.js

```diff
@@ -347,6 +411,21 @@ function ToggleStops()
     }
 }
 
+function ToggleAlerts()
+{
+    MapClick();

```

Comment:
If this is going to be used here can we rename it to something like `ToggleMapControls`